Content.Client/UserInterface/Systems/Chat/ChatUIController.cs

```diff
@@ -128,6 +128,15 @@ public sealed class ChatUIController : UIController
     /// </summary>
     private const int SpeechBubbleCap = 4;
 
+    /// <summary>
+    ///     Channels in which the typing indicator should not be shown
+    /// </summary>
+    private const ChatSelectChannel HiddenIndicatorChannels =
+        ChatSelectChannel.Admin
+        | ChatSelectChannel.Console
+        | ChatSelectChannel.OOC
+        | ChatSelectChannel.Telepathic;
```

Comment:
There is already ChatSelectChannel.IsOOC() or some other function like that. Best to separate this and put it into a function inside that class to avoid creating even more sources of trust